doc/using/Synthesis.rst

```diff
@@ -98,6 +98,26 @@ simulation back-ends. Hence, available options for synthesis are the same as for
 
     $ ghdl --synth --std=08 --no-assert-cover my_unit
 
+.. option:: --assert-assumes
+
+  Treat all PSL assertions like PSL assumes. If this option is used, GHDL generates `assume` directives
+  instead each `assert` directive during synthesis. This is similar to the `-assert-assumes`
+  option of Yosys `read_verilog`<http://www.clifford.at/yosys/cmd_read_verilog.html>`_ command.
```

Comment:
I think it should be:

```
`read_verilog <http://www.clifford.at/yosys/cmd_read_verilog.html>`_ 
```

Note the space after `read_verilog`. Same below.

Reply:
Strange syntax. I will fix it.